### DISCUSSION TITLE
Compatibility with Domyos EB 520

### DISCUSSION THREAD

**Discussion by tfpereira** [cagnulein/qdomyos-zwift#1192] [General]
Hey,

First of all thanks a lot for all the work you're doing for the community!

So I'm considering buying this bike to work with zwift (apologies for the website in portuguese):
https://www.decathlon.pt/p/bicicleta-estatica-bike-520-autoalimentada-e-conectada/_/R-p-331491

Are you aware if this Domyos EB 520 is compatible and with auto resistance? I don't see it listed on the Wiki under the compatible bikes hence the question, but It has E-Connected which is supossed to be compatible so I thought I'd first ask before I spend money in it 😅 

Thanks a lot!

**Comment by cagnulein** [comment]
yes it's compatible and also have auto resistance!

let me know if you have any further questions!

**Comment by Charly-T-Charly** [comment]
Hi Cagnulein,

I bought the Domyos EB520 bike (not the new version EB 520 B). 
It works correctly with the QZ app except for the resistance which is not controlled by QZ. 

Changing the resistance on QZ or in the applications (Zwift, Rouvy, MyWhoosh, ...) does not change the bike's resistance. 

You will find the debug attached. 
[debug-Mon_May_12_21_14_17_2025.log](https://github.com/user-attachments/files/20174380/debug-Mon_May_12_21_14_17_2025.log)

Another post presents an identical problem : https://github.com/cagnulein/qdomyos-zwift/issues/3433 

Do you know what is blocking it ? 

Charly

**Reply by cagnulein** [reply]
@Charly-T-Charly hi join the beta from here https://play.google.com/apps/testing/org.cagnulen.qdomyoszwift and update the app from the store 

Let me know!

**Reply by Charly-T-Charly** [reply]
Thanks Cagnulein, I will test this afternoon (2.18.24)

**Reply by gdassas** [reply]
Hello Charly, did it work ? 
I bought the Eb 900b and tough it would work too but doesn't seems that easy ^^
(eveything work apart the auto resistance, and sometimes zwift eventually lock it, how do i log the debug ?)

**Reply by cagnulein** [reply]
@gdassas debug instructions https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F

**Reply by gdassas** [reply]
thank you, i'll keep you tuned soon (not before this weekend hum)

**Reply by gdassas** [reply]
Hum i can't acces that menu, it's always behind android buttons' bar
![Screenshot_2025-08-03-16-17-46-230_org.cagnulen.qdomyoszwift.jpg](https://github.com/user-attachments/assets/1060b1aa-72b4-4efb-974e-9066acb62234)

Or if i put the phone in landscape, the toggle button then is behind it :(

![Screenshot_2025-08-03-16-19-36-649_org.cagnulen.qdomyoszwift.jpg](https://github.com/user-attachments/assets/ed6dc0bf-25c1-452a-a939-2d65b2d55dfc)

My bad, opening in landscape modify in portrait...

**Comment by cagnulein** [comment]
Join the beta, already fixed
https://play.google.com/apps/testing/org.cagnulen.qdomyoszwift
Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno dom 3 ago 2025 alle ore 16:21 gdassas ***@***.***>
ha scritto:

**Reply by gdassas** [reply]
hmm so now, debug is on,  CX File Explorer installed, but i can't acess the files... when i try to authorize it, the app block it to keep my privacy

**Comment by cagnulein** [comment]
Check under /documents/qz

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno dom 3 ago 2025 alle 16:58 gdassas ***@***.***> ha
scritto:

**Reply by gdassas** [reply]
nothing here :(

**Comment by cagnulein** [comment]
Android document under the experimental settings is enabled?

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno dom 3 ago 2025 alle 17:01 gdassas ***@***.***> ha
scritto:

**Reply by gdassas** [reply]
ah, hum no, ok back on the bike then

**Reply by gdassas** [reply]
![debug-Sun_Aug_3_17_01_46_2025.log](https://github.com/user-attachments/files/21566537/debug-Sun_Aug_3_17_01_46_2025.log)

![debug-Sun_Aug_3_17_05_19_2025.log](https://github.com/user-attachments/files/21566538/debug-Sun_Aug_3_17_05_19_2025.log)

Pfiou, i'm strugling for everything but here it is, anyway thank you for your time !

**Comment by cagnulein** [comment]
Thanks I will check it tomorrow

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno dom 3 ago 2025 alle 17:19 gdassas ***@***.***> ha
scritto:

**Reply by cagnulein** [reply]
@gdassas in the qz settings, bike options, domyos bike, enable the "Ignore FTMS" setting and restart QZ
Let me know

**Reply by gdassas** [reply]
That was it, it's all working fine now, thank you !!

**Comment by cagnulein** [comment]
Glad to hear it!


Il giorno lun 4 ago 2025 alle 18:56 gdassas ***@***.***> ha
scritto:
